suite-common/wallet-core/src/blockchain/blockchainThunks.ts

```diff
@@ -299,7 +299,11 @@ export const onBlockMinedThunk = createThunk(
     `${blockchainActionsPrefix}/onBlockMinedThunk`,
     (block: BlockchainBlock, { dispatch }) => {
         const symbol = block.coin.shortcut.toLowerCase();
-        if (isNetworkSymbol(symbol)) {
+
+        // Don't sync Solana because we emit a new block for Solana every 10 seconds.
+        // Solana accounts are updated via account subscription or also by the timer
+        // in syncAccountsWithBlockchainThunk.
+        if (isNetworkSymbol(symbol) && !['sol', 'dsol'].includes(symbol)) {
```

Comment:
NP, maybe use 
`const network = getNetwork(symbol.toLowerCase())` 
to avoid doing `sol || dsol`

Reply:
👍 I updated the commit - https://github.com/vacuumlabs/trezor-suite/pull/33/commits/7e34395d97272b6a2a49e11b2c28f36a97a101ba.